backend/examples/user_custom_properties/default/schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "",
+  "description": "",
+  "type": "object",
+  "required": [],
+  "properties": {
+    "tagline": {
+      "type": "string",
+      "title": "Tagline"
+    },
+    "localization": {
+      "type": "string",
+      "title": "Localization"
+    },
+    "hobbies": {
+      "title": "Hobbies",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "website": {
+      "type": "string",
+      "title": "Personal Website"
+    },
+    "birthday": {
+      "type": "string",
+      "format": "date",
+      "title": "Birthday"
+    },
+    "telephone": {
+      "type": "string",
+      "title": "Telephone",
+      "minLength": 10
+    },
+    "languages": {
+      "title": "Languages",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "personal_page": {
+      "type": "string",
+      "title": "Personal Page"
+    }
+  }
+}
```

Comment:
Why we not use uppercase and not lowercase (Sometime we seen some uppercase

Reply:
parameter name vs value to show in ui (title).

Reply:
What do you mean?

Reply:
the value have first character in uppercase whereas for title whereas the parameter are all lowercase

Reply:
Ok for the answer